tidy3d/plugins/mode/solver.py

```diff
@@ -18,6 +18,8 @@
 TOL_EIGS = fp_eps
 # Tolerance for deciding on the matrix to be diagonal or tensorial
 TOL_TENSORIAL = 1e-6
+# shift target neff by this value, both rel and abs, whichever results in larger shift
+TARGET_SHIFT = -10 * fp_eps
```

Comment:
why negative?

Reply:
i think either positive or negative is fine. Just randomly picked one.

Reply:
Positive can be safer to have a positive n_eff?

Reply:
I don't really have any intuition about it